final_task/rss_reader/News.py

```diff
@@ -0,0 +1,58 @@
+from dataclasses import dataclass
+import datetime
+import logging
+
+module_logger = logging.getLogger("rss_reader.scripts.News")
+
+
+@dataclass
+class News:
+    feed: str
+    title: str
+    date: datetime.datetime
+    link: str
+    info_about_image: str
+    briefly_about_news: str
+    links_from_news: list
+
+    def get_json(self):
+        """
+            returns news in json format
+        """
+        logger = logging.getLogger("rss_reader.scripts.News.get_json")
+        logger.info("return news in json format")
+        data = {
+            "Feed": self.feed,
+            "Title": self.title,
+            "Date": str(self.date),
```

Comment:
A question:

Isn't it already a string? Do you really need this conversion?

Reply:
date is <class 'datetime.datetime'>